rs/rosetta-api/icp_ledger/ledger/scripts/dfx_deploy.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+# This script is supposed to be run only through Bazel: bazel run //rs/rosetta-api/icp_ledger/ledger/scripts:dfx_deploy
+# It deploys the icp_ledger_canister canister with the specified arguments
+
+# Check if necessary environment variables are set
+if [[ -z "$MINTER_ACCOUNT_ID" ]]; then
+    echo "Error: MINTER_ACCOUNT_ID environment variable is not set"
+    exit 1
+fi
+
+if [[ -z "$DEFAULT_ACCOUNT_ID" ]]; then
+    echo "Error: DEFAULT_ACCOUNT_ID environment variable is not set"
+    exit 1
+fi
+
+# Check if a local replica is running
+if ! dfx ping &>/dev/null; then
+    echo "Error: Local replica is not running"
+    exit 1
+fi
```

Comment:
Would it make sense to check for existence of `dfx`? I do have it installed and I didn't check what error you would see, but I could imagine you may only see the `Error: Local replica is not running` message, rather than some `dfx: Command not found`? On the other hand, I suppose the chance that you would have the `MINTER_ACCOUNT_ID` and `DEFAULT_ACCOUNT_ID` environment variables set and _not_ have `dfx` is quite slim, so maybe it doesn't make sense after all.

Reply:
I guess it doesn't hurt, so I added it.